j2k/src/org/jetbrains/kotlin/j2k/ExpressionConverter.kt

```diff
@@ -867,7 +912,8 @@ class DefaultExpressionConverter : JavaElementVisitor(), ExpressionConverter {
                 if (parameters.size == 1 && !isKotlinFunctionType) {
                     // for lambdas all parameters with types should be present
                     emptyList()
-                } else {
+                }
+                else {
```

Comment:
Please restore previous code formatting - actual `if-else` statements and spaces are against accepted conceptions.